File: Python/Наименьший нечетный элемент массива.py
from random import random, randint
min=0
min1=20
a=['']*20
c=0
k=0
for i in range (20):
    a[i]=randint(0,20)
    if a[i]%2!=0 or a[i]==0:
        min=a[i]
        if min<min1:
            min1=min
print(a)
print(min1)

File: C++/Наименьший нечетный элемент массива.cpp
#include<iostream>
#include<cstdlib>
#include<ctime>
using namespace std;
int main(){
	srand(time(NULL));
	setlocale(LC_ALL, "Russian");
	int m[20];
	int min=0;
	int min1=20;
	int c=0;
	int k=0;
	for (int i=0; i<20; i++){
		m[i]=rand()%20+0;
		cout<<m[i]<<' ';
		if (m[i]%2!=0 || m[i]==0){
			min=m[i];
			if (min<min1){
				min1=min;
			}
		}
	}
	cout<<endl<<min1;
}
